extensions/amp-next-page/amp-next-page.md

```diff
@@ -25,134 +25,119 @@ limitations under the License.
+    <td>      
       <code>
         &lt;script async custom-element="amp-next-page"
-        src="https://cdn.ampproject.org/v0/amp-next-page-0.1.js">&lt;/script>
+        src="https://cdn.ampproject.org/v0/amp-next-page-1.0.js">&lt;/script>
       </code>
     </td>
   </tr>
   <tr>
-    <td>
-      <strong>
-        <a href="https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout">Supported Layouts</a>
-      </strong>
-    </td>
+    <td class="col-fourty"><strong><a href="https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout">Supported Layouts</a></strong></td>
     <td>N/A</td>
   </tr>
   <tr>
-    <td><strong>Examples</strong></td>
+    <td width="40%"><strong>Examples</strong></td>
     <td>See AMP By Example's <a href="https://amp.dev/documentation/examples/components/amp-next-page/">amp-next-page example</a>.</td>
   </tr>
 </table>
 
-[TOC]
+## Version notes
 
-## Behavior
+| Version | Description                                                                                                                                                                                                                                                                          |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| 1.0     | Support for an infinite number of page recommendations, reduced bundle size, improved API, support for amp-analytics, templated separators and footers, better handling of fixed elements.<br><br>API changes are breaking, please take a look at the migration section for details. |
+| 0.1     | Initial experimental implementation. Limited to three recommended documents                                                                                                                                                                                                          |
+
+## Usage
 
 Given a list of pages, `amp-next-page` tries to load them after the current
-document, providing an infinite-scroll type experience.
+document, providing an infinite-scroll experience.
 
 Pages should be specified in a JSON config loaded from a remote URL listed in
-the element `src`, or inlined as a `script` element child.
+the element `src`, or inlined as a `script` element child (or both).
 
 ```html
 <amp-next-page src="https://example.com/next-page-config.json"></amp-next-page>
 <!-- Or -->
 <amp-next-page>
   <script type="application/json">
-    {
-      "pages": ...
-    }
+    [
+      {
+        "url": ...,
+        "title": ...,
+        "image": ...
+      },
+      ...
+    ]
   </script>
 </amp-next-page>
 ```
 
 If loading the next document is successful it will be appended to the end of
```

Comment:
For the first question, no because you can just use both an `src` and an inline configuration in which case the inline configuration will load first then the remote configuration will be queries next (which is the same behavior as `next`)
Second question, depends on whether you have `deep-parsing` on or off, if it's on then it will load the inline articles from subsequent documents, otherwise it will limit itself to the host document's `<amp-next-page>` configuration